testing/trino-product-tests-launcher/src/main/java/io/trino/tests/product/launcher/env/EnvironmentOptions.java

```diff
@@ -55,7 +55,7 @@
     public String launcherBin;
 
     @Option(names = "--trino-jdk-version", paramLabel = "<trino-jdk-version>", description = "JDK to use for running Trino: ${COMPLETION-CANDIDATES} " + DEFAULT_VALUE, defaultValue = "ZULU_11")
-    public SupportedTrinoJdk jdkVersion = SupportedTrinoJdk.ZULU_11;
+    public SupportedTrinoJdk jdkVersion = SupportedTrinoJdk.ZULU_17;
```

Comment:
Please update docs.

Reply:
This PR does not switch Java requirement to 17 yet